tests/test_indexer.py

```diff
@@ -268,6 +268,10 @@ def test_subscription_notification_unsuccessful(self):
                          f"WARNING:.*:Failed notification for subscription {subscription_id}"
                          f" for bundle {bundle_id} with transaction id .+ Code: {error_response_code}")
 
+    @unittest.skip("WIP")
+    def test_index_when_multiple_indexes_using_alias(self):
```

Comment:
I think it would be good to add a simple test now that verifies that the alias is being created during indexing.
Then the test for an alias with multiple indexes can be added when that is the case, as you indicated here.

Reply:
Added